.github/workflows/ci.yaml

```diff
@@ -0,0 +1,53 @@
+name: CI
+
+on:
+  push:
```

Comment:
add `pull_request` as a trigger as well

Reply:
Do we have specific PR events we want to run on or just all PR events?

Reply:
The default is `opened`, `reopened`, and `synchronize` which should be enough for this and the changes have been pushed.